.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,22 @@
+name: "Check PR Title"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled, converted_to_draft, edited]
+
+jobs:
+  check-pr-title:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - name: Enforce conventional commit style
+        uses: realm/ci-actions/title-checker@main
+        with:
+          regex: '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test|ops){1}(\([\w\-\.]+\))?(!)?: .*'
+          error-hint: 'Invalid PR title. Make sure it follows the conventional commit specification (i.e. "<type>(<optional scope>): <description>") or add the no-title-validation label'
+          ignore-labels: 'no-title-validation'
+      - name: Enforce JIRA ticket in title
+        uses: realm/ci-actions/title-checker@main
+        with:
+          regex: 'VSCODE-[0-9]{1,5}$'
```

Comment:
It happens sometimes that we have tickets in one JIRA project that are touching multiple repos at the same time, should we extend this to allow all three (MONGOSH, COMPASS, VSCODE) or is the expectation now that we're supposed to open separate tickets when this happens?

Reply:
Yeah, I would vote for flexible behavior when no Jira tickets or multiple tickets are allowed.

Reply:
Enforcing for particular prefix is not super easy, so I figured I'll just skip the Jira ticket check for bot PRs.

Reply:
What about tiny PRs that do not have a corresponding ticket?

Reply:
You can add the `no-title-validation` label (or just merge with the check failing).